src/games/strategy/triplea/delegate/MoveDelegate.java

```diff
@@ -717,7 +669,7 @@ private void removeAirThatCantLand() {
    * Returns a map of unit -> transport. Tries to find transports to load all
    * units. If it can't succeed returns an empty Map.
    */
-  private static Map<Unit, Unit> mapTransportsToLoad(final Collection<Unit> units, final Collection<Unit> transports) {
+  public static Map<Unit, Unit> mapTransportsToLoad(final Collection<Unit> units, final Collection<Unit> transports) {
```

Comment:
Change to public so it can be called directly outside of this class as the previous method was removed.